technical-reference/reference-isolated-pools/risk-fund-and-shortfall/RiskFundV2.md

```diff
@@ -0,0 +1,80 @@
+# RiskFundV2
+
+Contract with basic features to track/hold different assets for different Comptrollers.
+
+# Solidity API
+
+### sweepToken
+
+Function to sweep baseAsset for pool, Tokens are sent to address(to)
```

Comment:
```suggestion
Function to sweep baseAsset for pool. Tokens are sent to address(to)
```

Reply:
nit: could ignore this since it is autogenerated but I think it is a bit odd to have a letter capitalized after a common that isn't a proper noun